client/src/components/Form/components/formSections/BehandlerSection.tsx

```diff
@@ -95,6 +96,11 @@ const BehandlerSection = ({ section, setFormState, formState, errors }: Behandle
                 }}
                 feil={errors.get('behandletDato')?.feilmelding}
             />
+            {formState.behandletDato != undefined &&
+            <div style={{ marginTop: '-0.5rem', marginBottom: '2rem' }}>
+                Dato valgt: {moment(formState.behandletDato).format("Do MMMM YYYY")}
+            </div>
+            }
```

Comment:
Regner med du bruker intellij. Da kjører den nok ikke noe formattering automatisk on-save. kjør `npm run format`, så formatterer den alle filer for deg.

Ville puttet dette inni komponenten `Datepicker` så slipper du å replikere dette hver gang du skal bruke det. Normaltekst wrappes rundt vanlig "paragraf"-tekst for å få default nav-styling.
```suggestion
            {!!formState.behandletDato && (
                <Normaltekst>
                    Dato valgt: {dayjs(formState.behandletDato).format("Do MMMM YYYY")}
                </Normaltekst>
            )}
```

Reply:
Takk!